demo/README.md

```diff
@@ -1,44 +1,59 @@
-# demo
+# Pelorus Demo
```

Comment:
Should we add this to the read the docs online documentation? I feel like a yes, because it is easier to run then the one we have today there